src/components/AvailableParkingSlotsView.tsx

```diff
@@ -41,15 +49,42 @@ export const AvailableParkingSlotsView: React.FC<IAvailableSlotsViewProps> = ({
     );
   }
 
+  function handleBuildingChange(
```

Comment:
Above comment, shift this function to the top following the line spacing.

Reply:
Convert this function declaration into arrow function

Reply:
resolved